src/ert/analysis/_es_update.py

```diff
@@ -160,51 +162,45 @@ def _load_observations_and_responses(
     auto_scale_observations: Optional[List[ObservationGroups]],
     progress_callback: Callable[[AnalysisEvent], None],
 ) -> Tuple[
-    npt.NDArray[np.float_],
-    Tuple[
-        npt.NDArray[np.float_],
-        npt.NDArray[np.float_],
-        List[ObservationAndResponseSnapshot],
-    ],
+    pd.DataFrame,
+    List[ObservationAndResponseSnapshot],
 ]:
     try:
-        observations_and_responses = ensemble.get_observations_and_responses(
-            [*selected_observations], iens_active_index
+        observed_responses = ensemble.get_observations_and_responses(
+            list(selected_observations), iens_active_index
```

Comment:
Nitpicking, but is the explicit conversion to list necessary?

Reply:
I think it can be removed, just have to change it to iterable where the arg goes further